doc/api/stream.md

```diff
@@ -1481,7 +1483,22 @@ run().catch(console.error);
 rs.resume(); // Drain the stream.
 ```
 
+`stream.finished()` leaves dangling event listeners (in particular
+`'error'`, `'end'`, `'finish'` and `'close'`) after `callback` has been
+invoked. The reason for this is so that unexpected `'error'` events (due to
+not entirely correct stream implementations) does not cause unexpected crashes.
+If this is unwanted behaviour then the returned cleanup function needs to be
```

Comment:
```suggestion
If this is unwanted behavior, then the returned cleanup function needs to be
```

(Our style-guide specifies US spellings. I believe that was selected to maintain consistency with the existing docs, although I wasn't around for that decision, I don't think.)